scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -409,9 +409,16 @@ object Enumerator {
 @branch trait Importee extends Tree with Ref
 object Importee {
   @ast class Wildcard() extends Importee
-  @ast class Name(name: scala.meta.Name.Indeterminate) extends Importee
-  @ast class Rename(name: scala.meta.Name.Indeterminate, rename: scala.meta.Name.Indeterminate) extends Importee
-  @ast class Unimport(name: scala.meta.Name.Indeterminate) extends Importee
+  @ast class Name(name: scala.meta.Name) extends Importee {
```

Comment:
Additionally, can we expose a `Name(): Name = Name.Anonymous()` constructor? Structure should then also print out `Name()` instead of `Name.Anonymous()`.